app/views/idv/doc_auth/_back.html.erb

```diff
@@ -21,11 +21,10 @@ classes << local_assigns[:class] if local_assigns[:class]
   <div class="margin-top-5 padding-top-2 border-top border-primary-light">
     <% if local_assigns[:action] %>
       <%= button_to(
-        text,
         path,
         method: :put,
         class: [*classes, 'usa-button usa-button--unstyled']
-      ) %>
+      ) {'‹ ' + t('forms.buttons.back')} %>
```

Comment:
per https://github.com/18F/identity-idp/pull/5288, can we add some spaces inside the braces?

```suggestion
      ) { '‹ ' + t('forms.buttons.back') } %>
```